### PR TITLE
v1.22.1

### PR DESCRIPTION
**Pull request by JelteF**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -4,6 +4,19 @@ PgBouncer changelog
 PgBouncer 1.22.x
 ----------------
 
+**2024-02-28  -  PgBouncer 1.22.1  -  "It's summer in Bangalore"**
+
+- Fixes
+  * Fix an issue involving some clients that could introduce memory leaks,
```
Comment (emelsimsek) [inline NEWS.md:10]:
```suggestion
  * Fix an issue impacting some clients using `COPY FROM STDIN` queries which could cause memory leaks, performance issues and issues with `max_prepared_statements>0`
```

Comment (emelsimsek) [inline NEWS.md:10]:
```suggestion
  * Fix issues caused by some clients using `COPY FROM STDIN` queries that could introduce memory leaks, performance issues and issues with `max_prepared_statements>0`  ([#1025])
    (bug introduced in 1.21.0)
```

**Review by emelsimsek — APPROVED** [review]

File: NEWS.md (hunk 1 above)
Comment (eulerto) [reply NEWS.md:10]: Let me suggest a different sentence:

* Fix issues caused by some clients using `COPY FROM STDIN` queries that might introduce memory leak, performance regression and prepared statement misbehavior.

Comment (JelteF) [reply NEWS.md]: Thanks for the suggestions. I pushed @eulerto his suggestion, with a small change that splits it into two sentences to make it a bit easier to read.

**Review by eulerto — APPROVED** [review]

**Pull request merged**